### PR TITLE
WIP: Migrate glide to dep

### PR DESCRIPTION
**Pull request by donce**
From https://github.com/Masterminds/glide:
> The Go community now has the dep project to manage dependencies. Please consider trying to migrate from Glide to dep.

### CODE REVIEW

File: bin/travis_scripts/ensure_dep.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    OS_ARCH="darwin-amd64"
+fi
+
+dep --version
+if [ $? -eq 0 ]; then
+    echo "Dep found"
+else
+    # TODO: fetch frozen dep version
+    go get -u github.com/golang/dep/cmd/dep
+#    wget "https://github.com/Masterminds/glide/releases/download/$GLIDE_VERSION/glide-$GLIDE_VERSION-$OS_ARCH.tar.gz"
```
Comment (Waldz) [inline bin/travis_scripts/ensure_dep.sh]: Not needed

Comment (donce) [reply bin/travis_scripts/ensure_dep.sh]: What's not needed, freezing version? why?

**Comment by donce** [comment]
Tests fails due to missing header file. This seems to be fixed as described in https://github.com/ethereumproject/go-ethereum/issues/237, but it'll be available only in next release 1.11, so it might be worth to delay this migration :/

**Comment by zyfdegh** [comment]
Thank you, @donce Will you continue your work?

**Comment by zyfdegh** [comment]
@donce No, sorry. I've never used `dep`

**Comment by donce** [comment]
If anybody is up for it, feel free to do it :)

**Comment by Waldz** [comment]
We will probably migrate to VGO https://github.com/golang/go/wiki/vgo

**Pull request closed without merging**